keps/sig-windows/1981-windows-privileged-container-support/README.md

```diff
@@ -267,26 +271,33 @@ How will UX be reviewed, and by whom?
+
+However:
+
 - Several upstream changes are required to support this feature in Kubernetes, including changes to containerd, OCI, CRI, and kubelet. The identified changes include (see CRI and Kubelet Implementation Details below for more details on changes):
-    - Containerd: enabling host network mode for privileged containers and pods ([working prototype demo](https://drive.google.com/file/d/1WQO2NDX34Z1FPbW7jEymhcPMY4AZWQSE/view)). Prototype is done using containerd runtimehandler but this proposal is to use cri-api.
+  - Containerd: enabling host network mode for privileged containers and pods ([working prototype demo](https://drive.google.com/file/d/1WQO2NDX34Z1FPbW7jEymhcPMY4AZWQSE/view)). Prototype is done using containerd runtimehandler but this proposal is to use cri-api.
     - OCI spec: https://github.com/opencontainers/runtime-spec
-        - [TBD]
+      - Updates pending decisions made in this KEP regarding namings.
     - CRI-api:
-        - Adding a privileged field to the runtime spec and pass through to containers
-        - Pass security context and flag of runtime spec to podsandbox spec (not currently supported, open issue: https://github.com/kubernetes/kubernetes/issues/92963) 
-    - Kubelet: Pass privileged flag and windows security context to runtime spec and other PSP changes (see below).
-- There are risks that changes at each of these levels may not be supported. 
-    - If containerd changes are not supported, host network mode will not be enabled.This would restrict the scenarios that privileged containers would enable, as CNI plugins, network policy, etc. rely on host network mode to enable access to host network resources.
-    - If CRI changes to enable a privileged flag are not supported, there would be a less-ideal workaround via annotations in the pod container spec. 
-    - The CRI changes may make an annotation in the OCI spec until the OCI updates are included.
+      - Adding `WindowsPodSandboxConfig` and `WindowsSandboxSecurityContext` message
+      - Adding `host_process` flag to `WindowsContainerSecurityContext`
+      - Pass security context and flag of runtime spec to podsandbox spec (not currently supported, open issue: https://github.com/kubernetes/kubernetes/issues/92963)
+  - Kubelet: Pass host_process flag and windows security context options to runtime spec.
+- There are risks that changes at each of these levels may not be supported.
+  - If containerd changes are not supported, host network mode will not be enabled.This would restrict the scenarios that privileged containers would enable, as CNI plugins, network policy, etc. rely on host network mode to enable access to host network resources.
+  - If CRI changes to enable a privileged flag are not supported, there would be a less-ideal workaround via annotations in the pod container spec.
+  - The CRI changes may make an annotation in the OCI spec until the OCI updates are included.
+
+### PSP support
```

Comment:
non-blocking nit: retitle to `Pod Security Implications` since the section isn't exclusively (or even primarily) about PSP now